### DISCUSSION TITLE
Component usages - Twilio Alarms

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#1312] [Office hours] [answered]
The specific questions I have are...

- Right location for a menu component when it's an empty state?
- Currently, we are planning to do 2 products(Messaging Insights and Error logs) for the Alarms. We are wondering how this will scale when we have multiple products (Current design shows two items in the menu component)
- (See other questions in the sign-up sheet)

**Background**: We are planning to do usability testing on Twilio Alarms which allows customers to monitor issues in your applications and resolve them quickly. 

Stage of design: Exploration
Research: (See sign up)

Links to mockups or prototype: https://www.figma.com/file/3GaINOa9Epx7ENTZslfaWa/Alerts-Q1-2021?node-id=952%3A9

**Comment by gloriliale** [comment]
Thanks for posting this so that we can discuss the details in office hours @sang-yeo !

**Comment by nkrantz** [comment, marked as answer]
Hi @sang-yeo! Please find the recording from yesterday's office hours [here](https://www.loom.com/share/7e2e180aa092438fbc9ac570f710fb8a) (password is in #help-design-system on slack).
